dot/sync/test_helpers.go

```diff
@@ -88,3 +91,43 @@ func BuildBlock(t *testing.T, instance runtime.Instance, parent *types.Header, e
 		Body:   body,
 	}
 }
+
+const (
+	defaultMinPeers     = 1
+	defaultMaxPeers     = 5
+	testTimeout         = time.Second * 5
```

Comment:
nit have this defined locally for the relevant tests (`TestChainSync_sync_bootstrap_withWorkerError` and `TestChainSync_sync_bootstrap_withWorkerError_Integration`) since it should be rather specific to each test.

Reply:
Updated.